aptible/resource_replica.go

```diff
@@ -1,19 +1,23 @@
 package aptible
 
 import (
+	"context"
+	"fmt"
 	"log"
+	"net/http"
 	"strconv"
 
-	"github.com/aptible/go-deploy/aptible"
+	"github.com/aptible/aptible-api-go/aptibleapi"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
 func resourceReplica() *schema.Resource {
 	return &schema.Resource{
-		Create: resourceReplicaCreate, // POST
-		Read:   resourceReplicaRead,   // GET
-		Update: resourceReplicaUpdate, // PUT
-		Delete: resourceReplicaDelete, // DELETE
+		Create:        resourceReplicaCreate, // POST
+		Read:          resourceReplicaRead,   // GET
+		UpdateContext: resourceReplicaUpdate, // PUT
```

Comment:
Any reason you chose to update the `Update` function to use context but not any of the others?

Reply:
This is how it was previously so I didn't want to mess with it

Reply:
I'm talking about switching from `Update` to `UpdateContext` and the [function definition changing](https://github.com/aptible/terraform-provider-aptible/pull/111/files#diff-244bf4fd57dbae2fdb736de87ca536f35d8867b93f364d8d4d6011a7463c1004R207) to use `context` and `diag` in this PR. I'm all for using the new `Context` functions but we should probably stick to one or the other on each resource.

Reply:
Ah, right.  That's because the database resource uses it and I just copied/pasted since the two functions are very similar: https://github.com/aptible/terraform-provider-aptible/blob/master/aptible/resource_database.go#L165